src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,65 @@
+import { Filter } from '../../types/Filters';
+import cn from 'classnames';
+
+type Props = {
+  countActiveTodos: number;
+  filter: Filter;
+  setFilter: (filter: Filter) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  countActiveTodos,
+  filter,
+  setFilter,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${countActiveTodos} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filter === Filter.All,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilter(Filter.All)}
+        >
+          {Filter.All}
+        </a>
```

Comment:
You can convert your enum into an array with `Object.keys/values/entries` and map through it to generate links